packages/@o3r/workspace/schematics/application/index.ts

```diff
@@ -83,6 +83,7 @@ function generateApplicationFn(options: NgGenerateApplicationSchema): Rule {
         ...Object.entries(extendedOptions).reduce((acc, [key, value]) => (angularOptions.includes(key) ? {...acc, [key]: value} : acc), {}),
         name: cleanName,
         projectRoot,
+        standalone: false,
```

Comment:
IMO the fix should be on the schematic to add `prefersReducedMotion` if it's the only part that does not work with standalone application.

Else we should not expose a `standalone` option in the schema.

Reply:
I was wrong when I thought we do not support standalone apps. We should fix the generator and add a test to make sure the content is correct.
Closing this PR.